res/manual/src/translate.md

```diff
@@ -101,3 +130,49 @@ Access the `completion` transformation via the `translate` command, e.g.
 ``` -->
 However, keep in mind that the original program must be tight for the models of the completion to coincide with the stable models of the program!
 This property is not checked automatically during translation.
+
+#### Simplifications
+
+As mentioned earlier, when using the `mu` translation, regular rules are transformed as `nu`, while
+the remaining rules use the `tau*` translation.
+However, it is still desirable to simplify the formulas produced by `tau*`,
+particularly if a rule is mostly regular.
+`anthem` employs three collections of simplifications for this purpose:
+
+* `intuitionistic` simplifications can be safely applied to the result of `tau*`, `nu`, `γ`, or `completion`;
+* `ht` simplifications can be similarly applied, and contain some transformations that are not intuitionistically valid;
+* `classical` simplifications should only be applied to the result of `γ`, or `completion`, this set of simplifications is an extension of `ht` just as `ht` extends `intuitionistic`.
+
+
+The `simplify` command allows users to apply any of the preceding portfolios of simplifications to a target language theory.
+For example, let `theory.spec` contain the formulas
+```
+    forall I J V1 (exists I1$i J1$i (V1 = I1$i * J1$i and I1$i = I and J1$i = J) and (exists Z Z1 (Z = I and Z1 = 1 and Z > Z1) and exists Z Z1 (Z = J and Z1 = 1 and Z > Z1)) -> composite(V1)).
+    forall I V1 (V1 = I and (exists Z Z1 (Z = I and exists I$i J$i K$i (I$i = 2 and J$i = n and Z1 = K$i and I$i <= K$i <= J$i) and Z = Z1) and exists Z (Z = I and not composite(Z))) -> prime(V1)).
+```
+Then, the command
+```
+    anthem simplify theory.spec --portfolio ht --strategy fixpoint
+```
+produces the ht-equivalent theory
+```
+    forall I J V1 (exists I1$i J1$i (V1 = I1$i * J1$i and I1$i = I and J1$i = J) and (I > 1 and J > 1) -> composite(V1)).
+    forall I V1 (V1 = I and (exists J$i K$i (J$i = n and I = K$i and (2 <= K$i and K$i <= J$i)) and (composite(I) -> #false)) -> prime(V1)).
+```
+
+Internally, `anthem` uses these simplifications within verification tasks unless the `--no-simplify` flag is added.
+The theories produced by `tau*` or `mu` are subjected to the `ht` simplification portfolio prior to applying `gamma` or `completion`.
+
+
+
+###### Intuitionistic Simplifications
+
+The majority of our simplifications are intuitionistically equivalent transformations from one formula `F` to another formula `G` (denoted `F => G`).
+These include identities (e.g. `F & true => F`),
+annihilations (e.g. `F & false => false`),
+and idempotences (e.g. `F & F => F`).
```

Comment:
We should list all available simplifications as a reference here -- or refer to the API documentation.